### ISSUE TITLE
Don't use `Views` as the default component folder in Rails

### ISSUE THREAD

**Issue by alexandreruban** [joeldrapper/phlex#180]
After working on documenting the Rails generator for Phlex components in https://github.com/joeldrapper/phlex/pull/178, I realized we still use the `app/views` folder as the default home for Phlex components.

I think it's not very practical, because `app/views` is already used for `.html.erb` templates in Rails. 

Would it make sense to use something else? The View Components gem uses `app/components` by default. Maybe we could use `app/phlex_components`?

**Comment by joeldrapper** [comment]
I changed it to `app/views` because I see Phlex as a complete replacement for ActionView. While migrating from ActionView, it would be okay to have some ERB and some Phlex in the same folder. For example, if I had converted my Articles#show view to Phlex but not my Articles#index, the folder would look like this.

```
app/views/articles/index.html.erg
app/views/articles/show.rb
```

I actually think it might be a good idea to rename `Phlex::Component` to `Phlex::View`. A lot of people who use ViewComponents don't realise they can use them for everything: pages, layout and partials.
